src/Meta.jl

```diff
@@ -0,0 +1,34 @@
+include("libraryfuncdictionary.jl")
+
+input_manipulator_funcs = Dict(
+    :dummy => Dict(
+        # :dummy => i->[ x + 1 for x in i]
+    )
+)
+
+output_manipulator_funcs = Dict(
+    :dummy => Dict(
+        # :dummy => i -> i + 1
+    )
+)
+
+for (name,funcs) in libraryfunctiondictionary
+    name_caps = Symbol( "Lib" * uppercasefirst(string(name)))
+    func_calls = Any[]
+    name_string = string(name) * ".lib"
+    for i in funcs
+        if i[1] == "g"
+            func_name = i[2]
+            symb = Symbol(func_name)
+            input_manipulator = haskey(input_manipulator_funcs,name) && haskey(input_manipulator_funcs[name],symb) ? input_manipulator_funcs[name][symb] : identity
```

Comment:
These lines are way to full. I'd suggest to choose shorter variable names. Also could use Dict::get, like so (untested), or perhaps replace the `?:` operator by a plain `if`?
```suggestion
            input_manipulator = haskey(input_manipulator_funcs,name) ? get(input_manipulator_funcs[name], symb, identity) : identity
```